docs/artifacts.md

```diff
@@ -0,0 +1,98 @@
+Artifact Versioning and Tagging (1.0.0)
+=======================================
+
+Artifact Name Composition
+-------------------------
+
+### Filenames
+
+|artifact name|@|semantic version|.|file extension|
+|:-:|:-:|:-:|:-:|:-:|
+|electron-bootloader|@|1.0.1+lto.debug.jtag|.|bin|
+
+**E.g.** `electron-bootloader@1.0.1+lto.debug.jtag.bin`
+
+
+#### Filename Composition
+
+- Artifact Name
+  - (Platform Family Name (_none_) | Platform Name (`electron`))
```

Comment:
Should we add a list of options here?

Reply:
There is a great list offered by our release artifacts. I believe a list of options inline here would only serve to muddy the explanation of the example.